com.unity.template-hd/Assets/SampleSceneAssets/Particles/FallingLeaves.vfx

```diff
@@ -540,7 +540,7 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   m_UIIgnoredErrors: []
-  m_Parent: {fileID: 114350483966674976}
+  m_Parent: {fileID: 0}
```

Comment:
@arttu-peltonen  Looks like various FX have lose a fileID, wondering if the FX are still working with your change? poking @JMargevics  maybe Jans you could check if VFX is still ok in the template? thanks

Reply:
yeah I'm not sure why this happened - I tried reverting these but when I save the project, it happens again. I still see falling leaves & flying butterflies so it looks like it's fine.

Reply:
I checked all VFX still work (Butterflies, dust, leaves)